core/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -44,6 +47,9 @@
 
 public class InternalSettingsPreparer {
 
+    private static Logger logger = ESLoggerFactory.getLogger("settings");
+    private static DeprecationLogger deprecationLogger = new DeprecationLogger(logger);
```

Comment:
Also I wonder if these loggers should be initialized in the one place they are used?